irondb/irondb.py

```diff
@@ -88,8 +90,9 @@ def fetch(self, start_time, end_time):
                 for i in range(0, min(urls.host_count, tries)):
                     try:
                         self.fetched = False
-                        d = requests.post(urls.series_multi, json = params, headers = self.headers, timeout=((self.connection_timeout / 1000), (self.timeout / 1000)))
-                        if d.headers['content-type'] == 'application/json':
+                        d = requests.post(urls.series_multi, json = params, headers = self.headers,
+                                          timeout=((self.connection_timeout / 1000), (self.timeout / 1000)))
+                        if not 'content-type' in d.headers or d.headers['content-type'] == 'application/json':
```

Comment:
If the `content-type` header is missing, isn't that an error?? Should always be present AFAIK.

Reply:
Rearranged this to make it more clear